src/IdentityServer4/src/Configuration/IdentityServerApplicationBuilderExtensions.cs

```diff
@@ -47,15 +47,15 @@ public static IApplicationBuilder UseIdentityServer(this IApplicationBuilder app
             return app;
         }
 
-        internal static void Validate(this IApplicationBuilder app)
+        internal static void Validate(this IServiceProvider provider)
```

Comment:
Moved this onto `IServiceProvider` so it could be shared with `MapIdentityServer`.